src/main/java/com/spaceclub/user/domain/User.java

```diff
@@ -38,20 +43,27 @@ public class User {
     @Lob
     private String refreshToken;
 
+    @OneToMany(mappedBy = "user")
+    private List<EventUser> events;
+
     @Builder
     private User(
+            Long userId,
             String nickname,
             String phoneNumber,
             String oauthUserName,
             Provider provider,
             Email email,
-            String refreshToken
+            String refreshToken,
+            List<EventUser> events
     ) {
+        this.userid= userId;
         this.requiredInfo = new RequiredInfo(nickname, new PhoneNumber(phoneNumber));
         this.oauthUserName = oauthUserName;
         this.provider = provider;
         this.email = email;
         this.refreshToken = refreshToken;
+        this.events = events;
```

Comment:
이러면 얕은복사가 일어나서 List에 예상치 못한 변경이 생길 수 있어요
깊은복사로 넘기시는건 어떠신가요?
```suggestion
        this.events = new ArrayList<>(events);
```

Reply:
깊은 복사로 넘기는 메서드는 list.copyof라는 메서드가 있는데 이걸로 수정할게요